src/integrationTest/java/uk/gov/hmcts/reform/pip/subscription/management/utils/IntegrationBasicTestBase.java

```diff
@@ -3,7 +3,9 @@
 import org.springframework.boot.test.mock.mockito.MockBean;
 import uk.gov.hmcts.reform.pip.subscription.management.repository.SubscriptionRepository;
 
-public class IntegrationBasicTestBase {
+public class IntegrationBasicTestBase extends IntegrationTestBase {
```

Comment:
Do we need to extend IntegrationBasicTestBase from IntegrationTestBase? I don't think those tests using the basic profile need to call out to external services.